setup.py

```diff
@@ -28,9 +28,9 @@
 with open('LONG_DESCRIPTION.rst') as f:
     long_description = f.read()
 
-# Don't import stripe module here, since deps may not be installed
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'stripe'))
-from version import VERSION
+about = {}
+with open(os.path.join('stripe', 'version.py')) as f:
+    exec(f.read(), about)
```

Comment:
This is slightly cleaner than importing, and removes a `flake8` warning about imports needing to be grouped at the top of the file. (Note that we `setup.py` is not automatically validated by flake8 currently.)

Reply:
Kind of a nit again, but could change `about` to something a little more descriptive like ` version_contents`?

Love the same though. The old code is tremendously opaque by comparison.

Reply:
👍